packages/payment-proxy-client/src/App.tsx

```diff
@@ -438,9 +456,10 @@ export default function App() {
                           } as React.CSSProperties
                         }
                       >
-                        Pay & Download
+                        Pay {prettyPrintFIL(selectedFile.size * costPerByte)} &
```

Comment:
This button is starting to take on a lot of work, not only does it show progress but it also shows the cost of the files now.